server/workflow.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/mattermost/mattermost-plugin-workflow-client/workflowclient"
+)
+
+type WorkflowTrigger struct {
```

Comment:
Is it really needed, the type? `AddTrigger(filters SubscriptionFilters` seems more-intention-revealing, 0/5

Reply:
It did originally have other fields. It is also a separate entity from the filters themselves. If we need to store other data about the workflow trigger in the store, we would put it here.